docs/tutorials/word-tutorial.md

```diff
@@ -738,15 +725,15 @@ Complete the following steps to define the image that you'll insert into the doc
 1. Add the following function to the end of the file.
 
     ```js
-    function insertTable() {
-        Word.run(function (context) {
+    async function insertTable() {
+        await Word.run(async (context) => {
 
             // TODO1: Queue commands to get a reference to the paragraph
             //        that will proceed the table.
 
             // TODO2: Queue commands to create a table and populate it with data.
 
-            return context.sync();
+            await context.sync();
         })
         .catch(function (error) {
```

Comment:
Should we keep these catch statements?

Reply:
There's no wrapper function like in the Script Lab snippets. We could rewrite the tutorial to have that, but I figured this practice here is good. It shows the action and error handling, which is needed in a complete app. In the other sample code blocks (outside of the tutorial), we're wanting to highlight the API in question, so the catch block only serves as a distraction.